Tests/ArgumentParserUnitTests/Snapshots/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*]
```

Comment:
I'd like to avoid this. Instead the code generator should not emit trailing whitespace and should generate a trailing newline. We can make this its own GitHub issue to resolve later to avoid making this even more complicated.

Reply:
We can keep it for now, I agree its annoying to deal with the editor messing up the file during development.